cryptodoc/src/13_pbkdf.rst

```diff
@@ -0,0 +1,17 @@
+Password Based Key Derivation Functions
+=======================================
+
+Argon2 (RFC 9106)
+-----------------
+
+Argon2 is a memory-hard function for password hashing.
+The implementation in Botan can be found in :srcref:`src/lib/pbkdf/argon2/argon2.cpp`
+and follows the standard [RFC9106]_.
+
+The maximum parallelism in Botan is more restricted than specified in [RFC9106]_.
+Botan allows a maximum of 128 threads.
+
+The maximum memory size in Botan is more restricted than specified in [RFC9106]_.
+Botan allows a maximum of :math:`2^{23}` KiB.
+
+The default parameter choice and tuning of parameters does not follow [RFC9106]_.
```

Comment:
`!! -v`?

Do you refer to [section 7.4](https://www.rfc-editor.org/rfc/rfc9106#section-7.4)?

This is actually sigificantly less secure than the suggested paramters. Memory param is defined as 128MiB vs. the 2GiB the RFC is proposing.

Shouldn't we at least add a suggestion to not use the default parameters but tune the algorithm manually? (The [TR02102](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR02102/BSI-TR-02102.pdf?__blob=publicationFile&v=8) vaguely suggests to ask an expert...)